kalliope/core/NeuronModule.py

```diff
@@ -171,11 +169,11 @@ def say(self, message):
             # save in kalliope memory the last tts message
             Cortex.save("kalliope_last_tts_message", tts_message)
 
-            # process the audio only if the no_voice flag is false
-            if self.no_voice:
-                logger.debug("[NeuronModule] no_voice is True, Kalliope is muted")
+            # process the audio only if the mute flag is false
+            if self.settings.start_options["mute"]:
```

Comment:
maybe "options" instead of "start" option. Not a big deal to leave it like this. 
Just in the future we could use this dict to store options on the fly that doesn't come from the starting.